ethcore/src/snapshot/account.rs

```diff
@@ -62,53 +63,50 @@ impl CodeState {
 
 // walk the account's storage trie, returning an RLP item containing the
 // account properties and the storage.
-pub fn to_fat_rlp(acc: &BasicAccount, acct_db: &AccountDB, used_code: &mut HashSet<H256>) -> Result<Bytes, Error> {
+pub fn to_fat_rlps(acc: &BasicAccount, acct_db: &AccountDB, used_code: &mut HashSet<H256>, max_storage_items: usize) -> Result<Vec<Bytes>, Error> {
```

Comment:
well, presumably it can only prevent an account from being much bigger than 4MB. given the docs i would have assumed that if the current chunk file is 13MB and we're dealing with an account with large storage, then 3MB of the storage would be placed in the current file (to get it up to 16MB). it sounds like that wouldn't be the case - rathe the 13MB file would be closed and a new file started with the first 4MB of the account.